.ddev/config.gum.yaml

```
@@ -0,0 +1,4 @@
+#ddev-generated
+webimage_extra_packages: [gum]
+web_environment:
+    - COLORTERM=${COLORTERM}
```

set `$COLORTERM` - as variable might not be set by host:
```suggestion
    - COLORTERM=truecolor
```